grpc-sys/build.rs

```diff
@@ -298,6 +298,8 @@ fn bindgen_grpc(mut config: bindgen::Builder, file_path: &PathBuf) {
         .clang_arg("-std=c++11")
         .rustfmt_bindings(true)
         .impl_debug(true)
+        .size_t_is_usize(true)
+        .disable_header_comment()
```

Comment:
I suggest to keep it so people may not try to edit the generated file by hand.

Reply:
The header comment now contains the bindgen version number, making the output less stable.

Reply:
Get it.